CRM/Utils/Date.php

```diff
@@ -342,8 +342,8 @@ public static function customFormat($dateString, $format = NULL, $dateParts = NU
     $fullWeekdayNames = self::getFullWeekdayNames();
     $abbrWeekdayNames = self::getAbbrWeekdayNames();
 
-    // backwards compatability with %D being the equivilant of %m/%d/%y
-    $format = str_replace('%D', '%m/%d/%y', $format);
+    // backwards compatibility with %D being the equivalent of %m/%d/%y
+    $format = str_replace('%D', '%m/%d/%y', ($format ?? ''));
```

Comment:
I wonder if in some cases casting to string would e better - eg. 
`$format = str_replace('%D', '%m/%d/%y', (string) $format)`

It's not important enough to change this PR but I think it would be slightly nicer in some cases